restful-api-with-data-mapper/Config.toml

```diff
@@ -0,0 +1,5 @@
+sheetsAccessToken = "<Google Sheets API access token>"
+spreadSheetId = "<Google spreadsheet ID>"
+
+# Optional
+# sheetName = "student-enrollments"
```

Comment:
For other samples we didn't commit the config toml. As a best practice it was added to .gitignore file. And README is updated with sample config.toml

Reply:
Sure, will remove this and add a README

Reply:
Fixed via [0961891](https://github.com/ballerina-guides/integration-samples/pull/27/commits/09618912461824f0bdd946319b01c6677c5f36d5)